apps/fishing-map/features/map/popups/ContextLayers.hooks.ts

```diff
@@ -68,17 +73,21 @@ export const useContextInteractions = () => {
       const datasetId = feature.datasetId
       const dataset = datasets.find((d) => d.id === datasetId)
       if (dataset) {
+        const dataview = dataviews.find((dataview) =>
+          dataview.datasets?.some((dataset) => dataset.id === datasetId)
+        )
         const areaName = feature.value || feature.title
         batch(() => {
           dispatch(setDownloadActivityAreaKey({ datasetId, areaId }))
+          dispatch(setDownloadActivityAreaDataview(dataview))
```

Comment:
We have to do the same in the [report download](https://github.com/GlobalFishingWatch/frontend/blob/develop/apps/fishing-map/features/reports/download/ReportDownload.tsx#L21), don't we?